src/calculateRentalCost.js

```diff
@@ -3,8 +3,25 @@
  *
  * @return {number}
  */
+
+const PRICE_FOR_DAY = 40;
+const LONG_TERM = 7;
+const LONG_TERM_DISCOUNT = 50;
+const SHORT_TERM = 3;
+const SHORT_TERM_DISCOUNT = 20;
+
 function calculateRentalCost(days) {
-  // write code here
+  const currentPrice = PRICE_FOR_DAY * days;
+
+  if (days >= LONG_TERM) {
+    return currentPrice - LONG_TERM_DISCOUNT;
```

Comment:
The long-term discount should be applied per day, not as a flat rate. Consider multiplying `LONG_TERM_DISCOUNT` by `days` before subtracting it from `currentPrice`.